core/client/app/templates/post-settings-menu.hbs

```diff
@@ -13,7 +13,12 @@
                 <label for="url">Post URL</label>
                 {{#if model.isDraft}}
                 <a class="post-preview-link" target="_blank" href="{{model.previewUrl}}">
-                    Preview<span class="icon-external"></span>
+                    Preview <i class="icon-external"></i>
+                </a>
+                {{/if}}
+                {{#if model.isPublished}}
```

Comment:
Yeah, was just writing that :)

On Fri, May 1, 2015 at 11:13 AM, JohnONolan <notifications@github.com>
wrote:

> In core/client/app/templates/post-settings-menu.hbs
> <https://github.com/TryGhost/Ghost/pull/5204#discussion_r29496123>:
>
> > @@ -13,7 +13,12 @@
> >                  <label for="url">Post URL</label>
> >                  {{#if model.isDraft}}
> >                  <a class="post-preview-link" target="_blank" href="{{model.previewUrl}}">
> > -                    Preview<span class="icon-external"></span>
> > +                    Preview <i class="icon-external"></i>
> > +                </a>
> > +                {{/if}}
> > +                {{#if model.isPublished}}
>
> I guess "isPublished" could be the default and "else" all other states get
> the preview - that seems relatively sane
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/TryGhost/Ghost/pull/5204/files#r29496123>.
>